Documentation/hubble.rst

```diff
@@ -0,0 +1,100 @@
+****************
+Hubble internals
+****************
+
+.. note:: This documentation section is targeted at developers who are
+          interested in contributing to Hubble. For this purpose, it describes
+          Hubble internals.
+
+.. note:: This documentation covers the Hubble server (sometimes referred as
+          "Hubble embedded") and Hubble-relay components but does not cover the
+          Hubble UI and CLI.
+
+Hubble builds on top of Cilium and eBPF to enable deep visibility into the
+communication and behavior of services as well as the networking infrastructure
+in a completely transparent manner.
+
+One of the design goals of Hubble is to achieve all of this at large scale.
```

Comment:
`all of this` at the start of a new paragraph carries little context

Reply:
Shall I merge this paragraph with the one above? Or simply `s/all of//` ?

Reply:
either way, maybe slightly more natural to combine them into a single paragraph.

Reply:
I've combined it into a single paragraph.